kitsune/messages/jinja2/messages/outbox.html

```diff
@@ -61,34 +46,33 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
                       {%- endif -%}
                     {%- if message.recipients_count > 1 -%}, ...{% endif %}
```

Comment:
I think we should move the `, ...` outside of the link for the single user we're showing here, the same way you're doing it for the group below.